spring-cloud-task-core/src/main/resources/org/springframework/cloud/task/schema-sqlserver.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE TASK_EXECUTION  (
+  TASK_EXECUTION_ID BIGINT  NOT NULL PRIMARY KEY ,
+  START_TIME DATETIME DEFAULT NULL ,
+  END_TIME DATETIME DEFAULT NULL ,
+  TASK_NAME  VARCHAR(100) ,
+  EXIT_CODE INTEGER ,
+  EXIT_MESSAGE VARCHAR(2500) ,
```

Comment:
This was meant as a more general statement (across all options), not just specifically this impl.